service-templates/apigw-lambda-svc/v1/instance_infrastructure/instance.tf

```diff
@@ -0,0 +1,147 @@
+################################################################################
```

Comment:
User preference - you can split up by files or by simple stanzas or both. This is just the format I use but please feel free to use whatever makes the most sense for yourselves. The key is to help logically group resources and keep supporting resources (data, locals, etc.) as close to where they are used as possible. Otherwise, flipping back and forth between multiple files can become a hassle and annoying. Again, completely user preference